metafeed-finder.js

```diff
@@ -9,6 +9,24 @@ const detectSsbNetworkErrorSeverity = require('ssb-network-errors')
 const { where, type, live, toPullStream } = require('ssb-db2/operators')
 const { validateMetafeedAnnounce } = require('ssb-meta-feeds/validate')
 
+class ReadyGate {
```

Comment:
I smell a module coming up :)

Reply:
Sigh, but not yet. I'm not too satisfied with this name, and with the whole implementation in general.